examples/gno.land/p/demo/grc/grc20/banker20.gno

```diff
@@ -7,16 +7,20 @@ import (
 	"gno.land/p/demo/ufmt"
 )
 
-// AdminToken implements a token factory with admin helpers.
+// TODO: Banker20 is the only struct
+// TODO: prefix nongrc20 with func Admin...
+// TODO: update doc
+
+// Banker20 implements a token banker with admin helpers.
 //
 // Warning: you should not expose this struct to enduser directly.
 //
 // It allows token administrators to call privileged helpers
 // like Mint, Burn, or any Transfer helpers by passing custom owners.
 //
-// You should initialize your token, then call AdminToken.SafeGRC20() to
+// You should initialize your token, then call Banker20.SafeGRC20() to
 // expose a safe instance to the endusers.
-type AdminToken struct {
+type Banker20 struct {
```

Comment:
NOTE: `Banker20` should have a `caller` variable (set to nil by default) for example which when you call `GRC20()` that return a `IGRC20`. force to use the `caller` variable.

This fix when you call from a realm the external method, or the internal method when you get the struct

```go
myfoo20 := foo20.SafeGRC20()
myfoo20.Transfer() // <-- std.PrevRealm could lead to error
```